index.js

```diff
@@ -44,8 +44,10 @@ exports.Districts = (province) => {
  * Sectors(); // => [] of all sectors found in Rwanda
  */
 exports.Sectors = (province, district) => {
-  if (province && district) {
-    return Object.keys(datafile[formatInput(province)][formatInput(district)]);
+  if (province && district && datafile[formatInput(province)]) {
+    return Object.keys(
+      datafile[formatInput(province)][formatInput(district)] || []
```

Comment:
Can't we throw an error instead of handling everything by returning an empty array as it might make people assume that the province or distric provided exists but without other details?

Reply:
Let me remove that optional array, and it will be throwing error automatically.

Reply:
I think it better to throw specific errors then the generic one to make those errors user friendly

Reply:
@oesukam you can now take a look at it.